syntax = "proto3";

package distributionSystemService;

service DistributionSystemService {
    rpc Ping(Empty) returns (PingResponse);
    rpc CollectData(Empty) returns (CollectedData);
}

message Empty {
  string num = 1;
}

message PingResponse {
  string server_address = 1;
  bool is_alive = 2;
}

message CollectedData{
  string someone = 1;
}
